ci-operator/config/openshift/origin/openshift-origin-master.yaml

```diff
@@ -33,6 +37,13 @@ images:
       as:
       - registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6
   to: tests
+- dockerfile_path: images/hello-openshift/Dockerfile.rhel
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.15-openshift-4.6:
+      as:
+      - registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6
+  to: hello-openshift
```

Comment:
Wonder if this is necessary since the `to: hello-openshift` target won't be consumed by any jobs, afaict :thinking: 
But, then again, im not sure what changes could be made instead.... @yselkowitz thoughts on who we can reach out to with questions here?

Reply:
Sounds like this will do the trick. nvmd.